src/modm/platform/clock/stm32/rcc.hpp.in

```diff
@@ -337,28 +371,36 @@ public:
 		Pll = RCC_CFGR_MCO2_1 | RCC_CFGR_MCO2_0,
 	};
 	%% endif
-%% elif target.family in ["l0", "l1", "l4", "l5", "g0", "g4"]
+%% elif target.family in ["l0", "l1", "l4", "l5", "g0", "g4", "u5"]
+%% set cfgr_mco="CFGR1" if target.family in ["u5"] else "CFGR"
 	enum class
 	ClockOutputSource : uint32_t
 	{
 		Disable = 0,
-		SystemClock = (1 << RCC_CFGR_MCOSEL_Pos), // SYSCLK
+		SystemClock = (0b0001 << RCC_{{cfgr_mco}}_MCOSEL_Pos), // SYSCLK
 	%% if target.family in ["l4", "l5"]
-		MultiSpeedInternalClock = (2 << RCC_CFGR_MCOSEL_Pos), // MSI
+		MultiSpeedInternalClock = (0b0010 << RCC_{{cfgr_mco}}_MCOSEL_Pos), // MSI
+	%% elif target.family in ["u5"]
+		MultiSpeedInternalClockS = (0b0010 << RCC_{{cfgr_mco}}_MCOSEL_Pos), // MSIS
+		Msis = MultiSpeedInternalClockS, // MISK
 	%% endif
 	%% if target.family in ["l0", "l1"]
-		InternalClock = (2 << RCC_CFGR_MCOSEL_Pos), // HSI16
-		MultiSpeedInternalClock = (3 << RCC_CFGR_MCOSEL_Pos), // MSI
+		InternalClock = (0b0010 << RCC_{{cfgr_mco}}_MCOSEL_Pos), // HSI16
+		MultiSpeedInternalClock = (0b0011 << RCC_{{cfgr_mco}}_MCOSEL_Pos), // MSI
 	%% else
-		InternalClock = (3 << RCC_CFGR_MCOSEL_Pos), // HSI16
+		InternalClock = (0b0011 << RCC_{{cfgr_mco}}_MCOSEL_Pos), // HSI16
 	%% endif
-		ExternalClock = (4 << RCC_CFGR_MCOSEL_Pos), // HSE
-		ExternalCrystal = (4 << RCC_CFGR_MCOSEL_Pos), // HSE
-		Pll = (5 << RCC_CFGR_MCOSEL_Pos), // Main PLL
-		LowSpeedInternalClock = (6 << RCC_CFGR_MCOSEL_Pos), // LSI
-		LowSpeedExternalClock = (7 << RCC_CFGR_MCOSEL_Pos), // LSE
-	%% if target.family == "l5"
-		Hsi48 = (8 << RCC_CFGR_MCOSEL_Pos), // HSI48
+		ExternalClock = (4 << RCC_{{cfgr_mco}}_MCOSEL_Pos), // HSE
+		ExternalCrystal = ExternalClock, // HSE
+		Pll = (5 << RCC_{{cfgr_mco}}_MCOSEL_Pos), // Main PLL
+		LowSpeedInternalClock = (6 << RCC_{{cfgr_mco}}_MCOSEL_Pos), // LSI
+		LowSpeedExternalClock = (7 << RCC_{{cfgr_mco}}_MCOSEL_Pos), // LSE
+	%% if target.family in ["l5"]
+		Hsi48 = (8 << RCC_{{cfgr_mco}}_MCOSEL_Pos), // HSI48
```

Comment:
Should we make the constants of this enum consistent and change all to binary notation?